src/main/kotlin/com/hibob/academy/employee_feedback_feature/service/FeedbackService.kt

```diff
@@ -0,0 +1,61 @@
+package com.hibob.academy.employee_feedback_feature.service
+
+import com.hibob.academy.employee_feedback_feature.dao.EmployeeRole
+import com.hibob.academy.employee_feedback_feature.dao.FeedbackDao
+import com.hibob.academy.employee_feedback_feature.dao.FeedbackRequest
+import com.hibob.academy.employee_feedback_feature.dao.FeedbackSubmission
+import jakarta.ws.rs.container.ContainerRequestContext
+import jakarta.ws.rs.core.Context
+import jakarta.ws.rs.core.Response
+import org.springframework.beans.factory.annotation.Autowired
+
+import org.springframework.stereotype.Component
+
+@Component
+class FeedbackService @Autowired constructor(private val feedbackDao: FeedbackDao) {
+    fun submitFeedback(@Context requestContext: ContainerRequestContext, newFeedback: FeedbackRequest): Response {
```

Comment:
you shouldn't handle requests in the service. Note that services can be used by other endpoints, not necessarily api's. For example - they can be used by an async job.
Requests should be handled at the controller level.

Reply:
Also not the @Context and ContainerRequestContext are not part of the springboot framework